### PR TITLE
ENG-271,ENG-272: Include default GitHub templates

### LINKED ISSUE #16: GH issue template file

### LINKED ISSUE #17: ENG-272: Include default GitHub feature request template file
- [A single issue template for your repository](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/manually-creating-a-single-issue-template-for-your-repository)
- [feature-request](https://github.com/othneildrew/Best-README-Template/blob/master/.github/ISSUE_TEMPLATE/feature-request---.md)

### PR DESCRIPTION
**Pull request by stefaniuk**
- Fixes #16 
- Fixes #17

Once this is merged we will get three options while creating an Issue, similarly as marked on the attached screenshot.

image

### CODE REVIEW

File: .github/SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Security
+
+NHS Digital takes security and the protection of private data extremely seriously. If you believe you have found a vulnerability or other issue which has compromised or could compromise the security of any of our systems and/or private data managed by our systems, please do not hesitate to contact us using the methods outlined below.
```
Comment (walteck) [inline .github/SECURITY.md:3]: Update to "NHS England"

**Review by walteck — CHANGES_REQUESTED** [review]
Couple of comments

File: .github/SECURITY.md (hunk 1 above)
Comment (stefaniuk) [reply .github/SECURITY.md:3]: thanks, upddated

File: .github/SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+If you wish to notify us of a vulnerability via email, please include detailed information on the nature of the vulnerability and any steps required to reproduce it.
+
+You can reach us at:
+
+- _[ product email address ]_
+- cybersecurity@nhs.net
+
+### NCSC
+
+You can send your report to the National Cyber Security Centre, who will assess your report and pass it on to NHS Digital if necessary.
```
[the remark this answers is not in the document]
Comment (stefaniuk) [reply .github/SECURITY.md:29]: thanks, upddated

**Review by walteck — APPROVED** [review]
LGTM

**Pull request merged**